solarwinds_apm/version.py

```diff
@@ -5,4 +5,4 @@
 # Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
 
 """Version of SolarWinds Custom-Distro for OpenTelemetry agents"""
-__version__ = "0.18.0.1"
+__version__ = "0.18.0.6"
```

Comment:
I did a few testreleases to create APM Python lambda layer from testpypi and do some testing there.